pkg/apis/extensions/v1beta1/types.go

```diff
@@ -397,6 +397,10 @@ type DeploymentStatus struct {
 	// +optional
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty" protobuf:"varint,3,opt,name=updatedReplicas"`
 
+	// Total number of ready pods targeted by this deployment.
+	// +optional
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
```

Comment:
I always manually add the proto tag and then run update-generated-protobuf.sh. Not sure if there is a script will add the tag.